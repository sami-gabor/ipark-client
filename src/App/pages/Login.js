import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import Button from 'react-bootstrap/Button';
import Form from 'react-bootstrap/Form';


class Login extends Component {
  render() {
    return (
      <div className="jumbotron d-flex align-items-center min-vh-100">
        <div className="container">
          <h1 className="text-center mb-5">Login</h1>
          <Form>
            <Form.Group controlId="formBasicEmail">
              <Form.Label>Email address</Form.Label>
              <Form.Control type="email" placeholder="Enter email" />
            </Form.Group>

            <Form.Group controlId="formBasicPassword">
              <Form.Label>Password</Form.Label>
              <Form.Control type="password" placeholder="Enter password" />
            </Form.Group>
            <p className="text-center">Not a member yet? Register <Link to="/register">here.</Link></p>
            <div className="text-center mt-5">
              <Link to={'/home'}>
                <Button variant="primary" type="submit">Login</Button>
              </Link>
            </div>
          </Form>
          test shell
        </div>
      </div>
    );
  }
}
export default Login;
