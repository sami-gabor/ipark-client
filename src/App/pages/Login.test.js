import { shallow } from 'enzyme';
import React from 'react';
import Login from './Login';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Login/>);
});

describe('<Login /> rendering', () => {
    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
})
