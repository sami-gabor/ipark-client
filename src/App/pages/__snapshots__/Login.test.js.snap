// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> rendering renders correctly 1`] = `ShallowWrapper {}`;

exports[`<Login /> rendering renders correctly 1`] = `ShallowWrapper {}`;
